### PR TITLE
feat: generate Swagger documentation

### PR DESCRIPTION
**Pull request by NishaSharma14**

### CODE REVIEW

**Comment by NishaSharma14** [comment]
image

File: .env.example [hunk 1]
```diff
@@ -97,10 +94,3 @@ NMRXIV_PROVIDER=NFDI4Chem
 NMRXIV_PROVIDER_URL=https://www.nfdi4chem.de/
 MEASUREMENT_TECHNIQUE=http://purl.obolibrary.org/obo/CHMO_0000613
```
[the remark this answers is not in the document]
Comment (NishaSharma14) [reply .env.example:95]: These are used by Noura for Bioschema

[the remark this answers is not in the document]
Comment (NishaSharma14) [reply .env.example:95]: these are used by Noura for Bischema

[the remark this answers is not in the document]
Comment (CS76) [reply .env.example:95]: Please move all bioschema code to a different pull request

**Pull request merged**